src/update_engine/download_action.cc

```diff
@@ -60,7 +60,15 @@ void DownloadAction::PerformAction() {
     processor_->ActionComplete(this, kActionCodeInstallDeviceOpenError);
     return;
   }
-
+  if (delta_performer_.get() &&
+      !delta_performer_->OpenKernel(
+          install_plan_.kernel_install_path.c_str())) {
+    LOG(ERROR) << "Unable to open kernel file "
+               << install_plan_.kernel_install_path.c_str();
```

Comment:
Yeah though this code is coming in via a revert rather than anything we wrote - I understand the desire for correctness, but this code is nasty enough that keeping it a little more stylistically consistent seems worthwhile